src/components/ActionBar/ActionBar.tsx

```diff
@@ -54,16 +73,34 @@ class ActionBar extends Component {
       tarball,
     };
 
-    const renderList = Object.keys(actionsMap).reduce((component, value, key) => {
+    const renderList = Object.keys(actionsMap).reduce((component: React.ReactElement[], value, key) => {
       const link = actionsMap[value];
       if (link && isURL(link)) {
-        const fab = <Fab size={'small'}>{ACTIONS[value]['icon']}</Fab>;
-        component.push(
-          // @ts-ignore
-          <Tooltip key={key} title={ACTIONS[value]['title']}>
-            <>{this.renderIconsWithLink(link, fab)}</>
-          </Tooltip>
-        );
+        const actionItem: Action = ACTIONS[value];
+        if (actionItem.handler) {
+          const fab = (
+            <Tooltip key={key} title={actionItem['title']}>
+              <Fab
+                /* eslint-disable react/jsx-no-bind */
+                onClick={() => {
+                  /* eslint-disable @typescript-eslint/no-non-null-assertion */
+                  actionItem.handler!(link);
```

Comment:
There is a check for `handler` above, I don't know why `eslint` does not understand that, so I used the null assertion to bypass the issue.